### PR TITLE
Making code style consistent for src/rustllvm (#38688)

### PR DESCRIPTION
**Pull request by karpinski**
Part of #38688

r? @brson

### CODE REVIEW

**Comment by rkruppe** [comment]
Whitespace-ignoring diff: https://github.com/rust-lang/rust/pull/38701/files?w=1

File: src/rustllvm/ArchiveWrapper.cpp [hunk 1]
```diff
@@ -76,174 +75,167 @@ typedef Archive::Child *LLVMRustArchiveChildRef;
...
-        // rustc_llvm currently doesn't use this error string, but it might be useful
-        // in the future, and in the mean time this tells LLVM that the error was
-        // not ignored and that it shouldn't abort the process.
-        LLVMRustSetLastError(toString(name_or_err.takeError()).c_str());
-        return NULL;
-    }
+  Expected<StringRef> name_or_err = child->getName();
+  if (!name_or_err) {
+    // rustc_llvm currently doesn't use this error string, but it might be
+    // useful
```
Comment (rkruppe) [inline src/rustllvm/ArchiveWrapper.cpp]: I'm a bit disappointed clang-format didn't re-wrap this. Mind fixing this up by hand?

**Comment by rkruppe** [comment]
Looks good to me, thanks a lot!

Will you also be tackling other bullet points from #38688, and if so, will it be in this PR or in a future PR?

File: src/rustllvm/ArchiveWrapper.cpp (hunk 1 above)
Comment (karpinski) [inline src/rustllvm/ArchiveWrapper.cpp]: Done

**Comment by karpinski** [comment]
@rkruppe I figured I would give someone else a chance to take a stab at the rest of them :)

**Comment by brson** [comment]
@bors r+

Thanks @karpinski !

**Pull request merged**